WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -170,6 +166,17 @@ - (void)viewWillDisappear:(BOOL)animated
     self.resultsController.delegate = nil;
 }
 
+- (void)didRotateFromInterfaceOrientation:(UIInterfaceOrientation)fromInterfaceOrientation
```

Comment:
Initially, I've tried using `(void)willAnimateRotationToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOrientation duration:(NSTimeInterval)duration` instead of `- (void)didRotateFromInterfaceOrientation:(UIInterfaceOrientation)fromInterfaceOrientation` however since the orientation has not been done yet, the width of the view would return the width before the orientation is done. Is there something I am missing here?